gpos/xml/scap/ssg/content/ssg-chainguard-gpos-ds.xml

```diff
@@ -188,9 +192,20 @@
                         For more information on inherited controls for containers see the DISA
                         Container Hardening Process Guide.</ns0:rationale>
                     <ns0:platform idref="cpe:/o:notapplicable" />
+                    <ns0:ident system="http://cyber.mil/legacy">V-263651</ns0:ident>
+                    <ns0:ident system="http://cyber.mil/legacy">SV-263651</ns0:ident>
+                    <ns0:ident system="http://cyber.mil/cci">CCI-000366</ns0:ident>
+                    <ns0:fixtext fixref="F-263651r1_fix">Configure the operating system to meet the requirement.</ns0:fixtext>
+                    <ns0:fix id="F-263651r1_fix" />
+                    <ns0:check system="http://oval.mitre.org/XMLSchema/oval-definitions-5">
+                        <ns0:check-content-ref href="stub-oval.xml" />
+                    </ns0:check>
```

Comment:
I'm not sure why these stub-oval checks got added; looking quickly, they seem to be added for the rules that we've marked ass not-applicable, and so shouldn't have a check, I think. (oscap warns about it when evaluating the xccdf against an image.)